hoomd/md/minimize/fire.py

```diff
@@ -0,0 +1,273 @@
+            :math:`[\\mathrm{force} / \\mathrm{mass}]`.
+        angmom_tol (float):
+            Angular momentum convergence criteria
+            :math:`[\\mathrm{energy} * \\mathrm{time}]`.
+        energy_tol (float):
+            Energy convergence criteria :math:`[\\mathrm{energy}]`.
+        min_steps_conv (int):
+            A minimum number of attempts before convergence criteria are
+            considered.
+        aniso (bool):
+            Whether to integrate rotational degrees of freedom (bool), default
+            None (autodetect).
+
+    `minimize.FIRE` is an `md.Integrator` that uses the Fast Inertial Relaxation
+    Engine (FIRE) algorithm to minimize the potential energy for a group of
+    particles while keeping all other particles fixed. This method is published
+    in `Bitzek, et. al., PRL, 2006
+    <http://dx.doi.org/10.1103/PhysRevLett.97.170201>`_.
+
+    At each time step, :math:`\\delta t`, the algorithm uses the supplied
+    integration methods to generate a x, v, and F, and then adjusts v according
+    to
+
+    .. math::
+
+        \\vec{v} = (1-\\alpha)\\vec{v} + \\alpha \\hat{F}|\\vec{v}|
+
+    where :math:`\\alpha` and :math:`\\delta t` are dynamically adaptive
+    quantities.  While a current search has been lowering the energy of system
+    for more than :math:`N_{min}` steps, :math:`\\alpha` is decreased by
+    :math:`\\alpha \\rightarrow \\alpha fdec_{\\alpha}` and :math:`\\delta t`
+    is increased by :math:`\\delta t \\rightarrow max(\\delta t \\cdot
+    finc_{dt}, \\ \\delta t_{max})`. If the energy of the system increases (or
+    stays the same), the velocity of the particles is set to 0, :math:`\\alpha
+    \\rightarrow \\ \\alpha_{start}` and :math:`\\delta t \\rightarrow \\delta t
+    \\cdot fdec_{\\alpha}`. The method converges when the force per
+    particle is below `force_tol`, the angular momentum is below `angmom_tol`
+    and the change in potential energy from one step to the next is below
+    `energy_tol`:
+
+    .. math::
+
+        \\frac{\\sum |F|}{N*\\sqrt{N_{dof}}} < \\mathrm{\\text{force_tol}}
+        \\;\\;, \\;\\ \\Delta \\frac{\\sum|E|}{N} <
+        \\mathrm{\\text{energy_tol}} \\;\\;, and \\;\\ \\frac{\\sum|L|}{N} <
+        \\mathrm{\\text{angmom_tol}}
+
+    where :math:`N_{\\mathrm{dof}}` is the number of degrees of freedom the
+    minimization is acting over. Any of the criterion can be effectively
+    disabled by setting the tolerance to a large number.
+
+    If the minimization acts on a subset of all the particles in the
+    system, the other particles will be kept frozen but will still interact
+    with the particles being moved.
+
+    Examples::
+
+        fire = md.minimize.FIRE(dt=0.05)
+        fire.force_tol = 1e-2
+        fire.energy_tol = 1e-7
+        fire.methods.append(methods.NVE(filter.All()))
+        sim.operations.integrator = fire
+        while not(fire.converged):
+           sim.run(100)
+
+    Examples::
```

Comment:
We have two `Examples` blocks now which doesn't make much sense, both examples should be in the block.